Unity/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "com.microsoft.vcrt.forwarders.140",

```

Comment:
Will this (and the added metas to this folder) be included in the NuGet package? If so, is that an issue?

Reply:
Definitely need to minimize impact on the nuget. Long term, we might want to actually remove the Unity support from the NuGet (that's a breaking change, so we'd need to bump the major version...).

Reply:
good question. i will investigate